config/environments/development.rb

```diff
@@ -5,6 +5,9 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Used to handle HTTP_X_WITH_SERVER_DATE header for server side datetime overwrite
+  config.middleware.use TimeTraveler
```

Comment:
Should also be in `deployed_development` and non-production environments.

Reply:
Added in the latest commit